packages/clay-pagination-bar/src/PaginationBarWithBasicItems.tsx

```diff
@@ -105,6 +105,8 @@ interface IProps extends React.ComponentProps<typeof PaginationBar> {
 	 * Use this property for i18n.
 	 */
 	labels?: {
+		pageDeltaPickerAriaLabel?: string;
```

Comment:
Hey @ilzamcmed  can you switch the name of the variable to `itemsPerPagePickerAriaLabel` to align with [this comment](https://github.com/liferay/clay/pull/5764#discussion_r1486105251)? The other variables use perpage instead of delta, so we should be consistent.

Reply:
Thanks @ethib137 
I think I missed this comment.  Everything is ok now.